scala-package/core/src/main/scala/org/apache/mxnet/api/java/JavaNDArray.scala

```diff
@@ -1,95 +1,185 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package org.apache.mxnet.api.java
 
 import org.apache.mxnet.NDArray
-import collection.JavaConverters
-import collection.JavaConverters._
-
-class JavaNDArray(val ndArray: NDArray) {
+import org.apache.mxnet.api.java.JavaDType.DType
 
-  /*
-  def this() = this(new NDArray())
-  (private[mxnet] val handle: NDArrayHandle,
-  val writable: Boolean = true,
-  addToCollector: Boolean = true)
-  */
-}
-
-object JavaNDArray{
+@AddJNDArrayAPIs(false)
+object JavaNDArray {
   implicit def fromNDArray(nd: NDArray): JavaNDArray = new JavaNDArray(nd)
 
-  implicit def toNDArray(jnd: JavaNDArray): NDArray = jnd.ndArray
-
-  val api = NDArray.api
+  implicit def toNDArray(jnd: JavaNDArray): NDArray = jnd.nd
 
   def waitall(): Unit = NDArray.waitall()
 
-  def onehotEncode(indices: JavaNDArray, out: JavaNDArray): JavaNDArray = NDArray.onehotEncode(indices, out)
+  def onehotEncode(indices: JavaNDArray, out: JavaNDArray): JavaNDArray
+  = NDArray.onehotEncode(indices, out)
 
-  def empty(shape: JavaShape, ctx: JavaContext, dtype: JavaDType.DType): JavaNDArray = NDArray.empty(shape, ctx, dtype)
-  def empty(shape:  Int *): JavaNDArray = NDArray.empty(shape: _*)
-  def empty(ctx: JavaContext, shape: Int *): NDArray = NDArray.empty(ctx, shape: _*)
+  def empty(shape: JavaShape, ctx: JavaContext, dtype: JavaDType.DType): JavaNDArray
+  = NDArray.empty(shape, ctx, dtype)
 
-  def zeros(shape: JavaShape, ctx: JavaContext, dtype: JavaDType.DType): JavaNDArray = NDArray.zeros(shape, ctx, dtype)
-  def zeros(shape: Int *): JavaNDArray = NDArray.zeros(shape: _*)
-  def zeros(ctx: JavaContext, shape: Int *): JavaNDArray = NDArray.zeros(ctx, shape: _*)
+  def empty(ctx: JavaContext, shape: Array[Int]): NDArray = NDArray.empty(ctx, shape: _*)
 
-  def ones(shape: JavaShape, ctx: JavaContext, dtype: JavaDType.DType): JavaNDArray = NDArray.ones(shape, ctx, dtype)
-  def ones(shape: Int *): JavaNDArray = NDArray.ones(shape: _*)
-  def ones(ctx: JavaContext, shape: Int *): JavaNDArray = NDArray.ones(ctx, shape: _*)
+  def zeros(shape: JavaShape, ctx: JavaContext, dtype: JavaDType.DType): JavaNDArray
+  = NDArray.zeros(shape, ctx, dtype)
 
-  def full(shape: JavaShape, value: Float, ctx: JavaContext): JavaNDArray = NDArray.full(shape, value, ctx)
+  def zeros(ctx: JavaContext, shape: Array[Int]): JavaNDArray = NDArray.zeros(ctx, shape: _*)
+
+  def ones(shape: JavaShape, ctx: JavaContext, dtype: JavaDType.DType): JavaNDArray
+  = NDArray.ones(shape, ctx, dtype)
+
+  def full(shape: JavaShape, value: Float, ctx: JavaContext): JavaNDArray
+  = NDArray.full(shape, value, ctx)
 
   def power(lhs: JavaNDArray, rhs: JavaNDArray): JavaNDArray = NDArray.power(lhs, rhs)
+
```

Comment:
There is no write or wrong here, I prefer grouping one liners that are all the same method but putting a break in them is equally valid. However, if you really want to make a lot of style changes it's better to do in a separate PR from functionality changes.